modules/jaeger-rn-technology-preview.adoc

```diff
@@ -0,0 +1,23 @@
+////
+Module included in the following assemblies:
+- rhbjaeger-release-notes.adoc
+////
+
+[id="jaeger-rn-tech-preview_{context}"]
+= Technology Preview
```

Comment:
Version 2.0 Technology Preview 1 - not sure if you wanted to distinguish it from other tech preview numbers that may come.

Reply:
So version 2.0 is not going to be the OTEL GA?

Reply:
Yes Jaeger 2.0 will eventually become the OTel GA - probably released with OSSM 2.1.